source/com.microsoft.tfs.core/rest_core/com/microsoft/visualstudio/services/webapi/model/ReferenceLinksDeserializer.java

```diff
@@ -16,7 +16,7 @@
 import com.fasterxml.jackson.databind.JsonDeserializer;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.microsoft.tfs.core.Messages;
-import com.microsoft.vss.client.core.utils.StringUtil;
```

Comment:
Are we sure that this change is correct? Is tfs.util.StringUtil.* a perfect match for vss.client.core.utils.StringUtil.*?